bskyweb/cmd/bskyweb/server.go

```diff
@@ -342,6 +343,42 @@ func (srv *Server) WebPost(c echo.Context) error {
 		}
 		data["imgThumbUrls"] = thumbUrls
 	}
+
+	// expand links in rich text back to full urls, replacing shortened urls in
+	// social card meta tags and the noscript output. this essentially reverses the effect
+	// of shortenLinks() in src/lib/strings/rich-text-manip.ts
+	if postView.Record != nil {
```

Comment:
this section should get refactored out in to a separate function; that will make testing easier. it should probably return a string, and maybe an error as well.

Reply:
as a style nit, the code in this block would be less-indented if the check was:

```go
if postView.Record == nil {
    return ""
}
```